operate/client/src/modules/components/Popover/ArrowPopover.tsx

```diff
@@ -63,7 +64,11 @@ const ArrowPopover: React.FC<Props> = ({
     refs: {floating, setFloating},
   } = useFloating({
     placement,
-    middleware: [...middlewareOptions, arrow({element: arrowElementRef})],
+    middleware: [
+      ...middlewareOptions,
+      shift({padding: {top: 47}}), // 47px = Operate app header height
```

Comment:
In my opinion we should start putting values like these into clearly named constants instead of using the numeric value and adding comments. What do you think?

Another instance where this could be beneficial are tests that test for number of instances that appear, for example. I feel like that way the code is a bit more organized and we avoid having "arbitrary" numbers that require comments to know what they are.

I don't think it's a problem, but mostly for keeping the codebase organized as it grows larger, so it's more of a suggestion.

Reply:
@vitorwtavares Good point, I will create a separate PR where I improve this.